docs/ehrql-new-tutorial-intro.md

```diff
@@ -228,14 +200,51 @@ and query that data instead of a backend.
 Each CSV represents a database table.
 Data Builder's CSV loader uses the filename of the CSV
 to determine which Data Builder table it represents.
-For example, a file named `patient_demographics.csv` is interpreted as the Data Builder `patient_demographics` table.
+For example, a file named `patients.csv` is interpreted as the Data Builder `patients` table.
 
 !!! todo
     Add information about loading CSVs elsewhere.
 
-#### Using Data Builder's command-line interface
+#### The OpenSAFELY CLI
+
+With the OpenSAFELY CLI,
+there are currently two ways of running the tutorial dataset definitions:
+
+* `opensafely exec` where the Data Builder configuration is specified at the command line
+* `opensafely run` where the Data Builder configuration is specified in the `project.yaml` file provided in `ehrql-tutorial-examples`
+
+We explain both of these below.
+
+!!! important
+    You only need to use one of these two ways,
+    though you are welcome to try both.
+    We will likely favour the `opensafely exec` route
+    for the tutorial in future
+    and relegate the use of `project.yaml` to the tutorial page that discusses `project.yaml`.
+
+    We are awaiting the improvements in
+    <https://github.com/opensafely-core/opensafely-cli/issues/149>
+    before doing so.
+
+#### Using the OpenSAFELY CLI to run a dataset definition via `opensafely exec`
+
+!!! warning
+    This requires:
+    * OpenSAFELY CLI version v1.32.1 or later
+      (run `opensafely upgrade` to upgrade)
+    * Data Builder Docker image v0.57.0 or later
+      (run `opensafely pull` to update if you already have
```

Comment:
This text is truncated.

Reply:
Thanks; now fixed, and updated since that command wouldn't work anyway right now.